core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -98,6 +98,34 @@
       .withPartitionPath("data_bucket=3") // easy way to set partition data for now
       .withRecordCount(1)
       .build();
+  static final DataFile FILE_PARTITION_0 = DataFiles.builder(SPEC)
+      .withPath("/path/to/data-0.parquet")
+      .withFileSizeInBytes(10)
+      .withPartitionPath("data_bucket=0")
+      .withPartition(TestHelpers.Row.of(0))
```

Comment:
You don't need both `withPartitionPath` and `withPartition`. The first one just parses the partition path to create the partition tuple. It's better to use `withPartition` instead (the other files in this class are quite a bit older).

Reply:
Done